pkgs/tools/text/wrap/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitHub, makeWrapper, courier-prime }:
+
+buildGoModule rec {
+  pname = "wrap";
+  version = "unstable-2021-04-13";
```

Comment:
I think the version can be 0.3.1, with the PR you submitted upstream used here with `fetchpatch` https://github.com/Wraparound/wrap/compare/v0.3.1...develop .

Reply:
We'd watch the fetchpatch pinned to a commit though, right? Not to `develop`?

Like https://github.com/Wraparound/wrap/compare/v0.3.1...b72c280b6eddba9ec7b3507c1f143eb28a85c9c1.patch

Reply:
Use https://github.com/Wraparound/wrap/commit/b72c280b6eddba9ec7b3507c1f143eb28a85c9c1.patch .

Reply:
Done!